tmail-backend/apps/postgres/src/main/java/com/linagora/tmail/james/app/PostgresTmailConfiguration.java

```diff
@@ -30,7 +30,7 @@ public record PostgresTmailConfiguration(ConfigurationPath configurationPath, Ja
                                          MailboxConfiguration mailboxConfiguration,
                                          BlobStoreConfiguration blobStoreConfiguration,
                                          SearchConfiguration searchConfiguration,
-                                         UsersRepositoryModuleChooser.Implementation usersRepositoryImplementation,
+                                         com.linagora.tmail.combined.identity.UsersRepositoryModuleChooser.Implementation usersRepositoryImplementation,
```

Comment:
You removed the previous dependency and you imported this one, so I think you can call UsersRepositoryModuleChooser directly